### ISSUE TITLE
Wrong ISO Code for Northern Cyprus

### ISSUE THREAD

**Issue by kajyr** [highcharts/highcharts#8944]
#### Expected behaviour

Northern Cyprus should have code `CY`, and `NC` should be reserved for New Caledonia

Link to ISO standard:

https://en.wikipedia.org/wiki/ISO_3166-2:CY
https://en.wikipedia.org/wiki/ISO_3166-2:NC

#### Actual behaviour

New Caledonia is not available and `NC` points to Northern Cyprus

#### Live demo with steps to reproduce
https://jsfiddle.net/kajyr/nwmtLgof/

#### Affected browser(s)
All browsers

**Comment by KacperMadej** [comment]
Hi @kajyr 

The change you are requesting is about `hc-a2` and in your demo you are using `hc-key`.

> `hc-key`: Unique hierarchical identifier for the area. Consistent across maps, and reflected in the naming convention of the maps. More detailed description below.

This cannot be changed to `cy` because the `cy` is already in use.

> 'hc-a2': Two letter abbreviation of area name. Usually same as postal code/ISO code, but not always. Not guaranteed to be unique within map.

This can be changed to from `NC` to `CY`, as duplicates are allowed and Northern Cyprus doesn't have it's own ISO code.

`New Caledonia` is a part of `France` on the map - https://jsfiddle.net/BlackLabel/nwmtLgof/5/ (zoom in to the New Caledonia location), so it doesn't have properties different than `France` has.

In short: are you requesting change of `hc-a2` for Northern Cyprus to `CY` because of name collision with New Caledonia in ISO code `NC`?

**Comment by jon-a-nygaard** [comment]
We have created a seperate repository for the map collection, and we are transfering the issues related to the map collection to this new repository.

Please continue the discussion and follow the future development of this issue at highcharts/map-collection-dist#26
